c++/bubble_sort.cpp

```diff
@@ -0,0 +1,36 @@
+#include <iostream>
+using namespace std;
+
+void bubbleSort(int arr[], int s) 
+{
+  for (int step = 0; step < (s-1); ++step) 
+  {
+    for (int i = 0; i < s - (step-1); ++i) 
```

Comment:
This is doesn't seem right

at `step = 0`, `i` will go until `s` which is array out of bounds it should be `s - step - 1`